kolibri/plugins/learn/assets/src/vue/search-widget/index.vue

```diff
@@ -237,6 +237,7 @@
     background-color: $core-bg-canvas
     position: fixed
```

Comment:
Didn't see that this was fixed, but I feel like this is something that should be avoided as often as possible.